<ui version="4.0" >
 <class>ReloginDialog</class>
 <widget class="QDialog" name="ReloginDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>输入用户名密码</string>
  </property>
  <property name="windowIcon" >
   <iconset>
    <normaloff>:/img/images/user.png</normaloff>:/img/images/user.png</iconset>
  </property>
  <property name="styleSheet" >
   <string notr="true" >font: 12pt "Sans Serif";</string>
  </property>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>20</y>
     <width>321</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title" >
    <string>请输入用户名密码</string>
   </property>
   <widget class="QLineEdit" name="txt_user" >
    <property name="geometry" >
     <rect>
      <x>100</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ok" >
    <property name="geometry" >
     <rect>
      <x>220</x>
      <y>30</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>确 定</string>
    </property>
    <property name="default" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_close" >
    <property name="geometry" >
     <rect>
      <x>220</x>
      <y>70</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>关 闭</string>
    </property>
    <property name="default" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_password" >
    <property name="geometry" >
     <rect>
      <x>100</x>
      <y>70</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="echoMode" >
     <enum>QLineEdit::PasswordEchoOnEdit</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>30</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>用户名：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>70</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string> 密  码：</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txt_user</tabstop>
  <tabstop>txt_password</tabstop>
  <tabstop>btn_ok</tabstop>
  <tabstop>btn_close</tabstop>
 </tabstops>
 <resources>
  <include location="img.qrc" />
 </resources>
 <connections/>
</ui>
